### PR TITLE
Add register metric

### PR DESCRIPTION
**Pull request by Louis-Dupont**
Still needs some work on the ReadMe

### CODE REVIEW

File: src/super_gradients/training/metrics/user_metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ <br/><br/>
+
+## Introduction
+This page demonstrates how you can register your own metrics, so that SuperGradients can access it with a name `str`, for
+example, when training from a recipe config.
+
+## Usage
+**In your python script**
+1. Define your PyTorch metric (`torchmetrics.Metric`) in the new module.
+2. Import the `@register` decorator
```
Comment (ofrimasad) [inline src/super_gradients/training/metrics/user_metrics/README.md:12]: @register_metric

**Review by ofrimasad — COMMENTED** [review]
minor comment

File: src/super_gradients/training/metrics/user_metrics/README.md (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/training/metrics/user_metrics/README.md]: Updated, there are also other changes in the ReadMe

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**